packages/components/src/utils/getPropsFrom.js

```diff
@@ -4,6 +4,7 @@ const NATIVE_PROPS = [
 	'className',
 	'id',
 	'name',
+	'target',
```

Comment:
could you update test getPropsFrom.test.js with this props to avoid regression later